documentation/modules/con-configuring-user-operator.adoc

```diff
@@ -32,6 +32,9 @@ For more details about resource request and limit configuration, see xref:assemb
 The `logging` property configures the logging of the User Operator.
 For more details, see xref:con-entity-operator-logging-{context}[].
 
+`secretPrefix`::
+The `secretPrefix` property can be used to add a prefix to the name of all Secrets created from the KafkaUser resource. e.g., `STRIMZI_SECRET_PREFIX=kafka-` would cause all secret names to start with `kafka-`, e.g. `kafka-my-user`.
```

Comment:
I think this is what failed the CI builds:

```
Replace 'e.g'. with 'for example, ':
documentation/modules/con-configuring-user-operator.adoc:36:The `secretPrefix` property can be used to add a prefix to the name of all Secrets created from the KafkaUser resource. e.g., `STRIMZI_SECRET_PREFIX=kafka-` would cause all secret names to start with `kafka-`, e.g. `kafka-my-user`.
ERROR: 1 docs problems found.
Makefile:130: recipe for target 'docu_check' failed
make: *** [docu_check] Error 1
```

Can you use _For example, _ instead of _e.g., _